hal/tls/mbedtls/tls_mbedtls.c

```diff
@@ -898,16 +904,10 @@ startRenegotiationIfRequired(TLSSocket self)
     if (Hal_getTimeInMs() <= self->lastRenegotiationTime + self->tlsConfig->renegotiationTimeInMs)
         return true;
 
-    raiseSecurityEvent(self->tlsConfig, TLS_SEC_EVT_INFO, TLS_EVENT_CODE_INF_SESSION_RENEGOTIATION, "Info: session renegotiation started", self);
-
-    if (TLSSocket_performHandshake(self) == false) {
-        DEBUG_PRINT("TLS", " renegotiation failed\n");
+    if (TLSSocket_performHandshake(self) == false)
         return false;
-    }
 
-    DEBUG_PRINT("TLS", " started renegotiation\n");
```

Comment:
Agreed removing that or at least fix ot to "completed renegotiation"

Reply:
Yess, moved both completed/failed in `TLSSocket_performHandshake` right after the related mbedtls call